app/build.gradle

```diff
@@ -106,6 +110,11 @@ dependencies {
     compile "com.squareup.retrofit2:converter-gson:$retrofitVersion"
     compile "com.google.code.gson:gson:2.7"
 
+    // RX
+    compile "io.reactivex.rxjava2:rxjava:2.0.4"
```

Comment:
We should consider defining version numbers as variables in the gradle like $retrofitVersion so it's easier to change. Why arent the rxjava and rxandroid version numbers different?